client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningGetResultsIT.java

```diff
@@ -90,6 +92,9 @@ public void createJobAndIndexResults() throws IOException {
         addBucketIndexRequest(time, true, bulkRequest);
         addRecordIndexRequest(time, true, bulkRequest);
 
+        // index some category results
+        addCategoriesIndexRequests(bulkRequest);
```

Comment:
I think we could perform this just in the get categories test. However, I suppose it doesn't really add to the execution time. Note that this setup is run once per test. Might be fun trying to see if there's a noticeable difference. :-)

Reply:
Good point! That call is only indexing 3 category results so I suspect the impact on execution time is low but I'll make the change regardless. Anything we can do to speed up the tests has to be a good thing...